### PR TITLE
javascript-data-model | Wonseop Lee

### PR DESCRIPTION
**Pull request by wonseop-lee**
image
image
image

### CODE REVIEW

File: javascript-data-model/main.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+var orderHistory = [
+  {
+    {
+      orderNumber = 11439416898772232,
+      orderPlaced = '8/4/2020',
+      total = 34.00,
+      shipTo = 'JS Masher'
+    },
+    [
```
Comment (BrettAlbright) [inline javascript-data-model/main.js]: you don't want to have another array here. you only need one array to contain all of the objects.

File: javascript-data-model/main.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
+/* exported orderHistory */
+
+var orderHistory = [
+  {
```
Comment (BrettAlbright) [inline javascript-data-model/main.js:4]: there is no need for another object here, you can put the order objects directly into the array:

```javascript
var orderHistory = [ {}, {}, {}]
```

File: javascript-data-model/main.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    ]
+  },
+  {
+    {
+      orderNumber = 11328831772648248,
+      orderPlaced = '7/3/2020',
+      total = 138.93,
+      shipTo = 'JS Masher'
+    },
+    [
```
Comment (BrettAlbright) [inline javascript-data-model/main.js]: it is good to use an array to separate the items here, but I would attach them to a property on the order object

```
orderNum:
orderPlaced:
items: [
{title:},
{title:}
]

```

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
👍

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**